app/src/main/java/org/feature/fox/coffee_counter/data/local/database/tables/Image.kt

```diff
@@ -0,0 +1,24 @@
+package org.feature.fox.coffee_counter.data.local.database.tables
+
+import androidx.room.Entity
+import androidx.room.ForeignKey
+import androidx.room.PrimaryKey
+
+@Entity(
+    tableName = "image",
+    foreignKeys = [ForeignKey(
+        entity = User::class,
+        parentColumns = arrayOf("id"),
+        childColumns = arrayOf("userId"),
+        onUpdate = ForeignKey.CASCADE,
+        onDelete = ForeignKey.CASCADE
+    )]
+)
+data class Image(
+    @PrimaryKey
+    val userId: String,
+
+    var encodedImage: String,
+
+    var timestamp: Long,
+)
```

Comment:
This should be an extra kotlin file for consistency reasons.

Reply:
Solved, as discussed in Discord.